### ISSUE TITLE
suggestion: create set official docker images

### ISSUE THREAD

**Issue by gavanderhoorn** [rdiankov/openrave#394]
Seeing as the installation of OpenRAVE on recent Linux distributions is complex (at least to get right, with specific versions of dependencies required, etc), it might be nice to provide a set of Docker images (through the hub). I've searched a bit and there are a few out there, but they are either not maintained, or are user contributed (which is fine).

An 'official' set of images would make setup of OpenRAVE a breeze, even if they are just base images with no 3D acceleration (which users would have to layer on themselves).

**Comment by Neon22** [comment]
The inkscape people were recently playing with something called Appimage which was working well for them...

**Comment by ziyan** [comment]
The problem with an official OpenRAVE docker image would be having to include adequate built-in graphics driver support, which will make the image very large.

**Comment by gavanderhoorn** [comment]
@ziyan: not necessarily. A base image that includes a pre-built version of OpenRAVE, but not any hw-specific graphics drivers would already be a huge timesaver.

**Comment by probonopd** [comment]
Have a look at the [Krita AppImage](https://krita.org/en/download/krita-desktop/); I think it uses the native OS graphics drivers intensively and seems to be working well.

**Comment by gavanderhoorn** [comment]
@probonopd: seems you are the author of AppImage?

**Comment by probonopd** [comment]
@gavanderhoorn yes, direct to me any questions or doubts you might have regarding it.

**Comment by gavanderhoorn** [comment]
I don't have any. I just thought I'd mention you are the author. Your comment made it look like you were "just another user".
